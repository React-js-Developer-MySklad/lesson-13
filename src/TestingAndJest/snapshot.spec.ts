describe('snapshot', () => {
  it('should work', () => {
    expect([122]).toMatchSnapshot();
  });
});
