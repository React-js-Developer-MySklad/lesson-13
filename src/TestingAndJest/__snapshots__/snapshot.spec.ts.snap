// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot should work 1`] = `
[
  122,
]
`;
